### PR TITLE
test: create bddtest structure

### LINKED ISSUE #5: create bddtest structure

### PR DESCRIPTION
**Pull request by fqutishat**
closes #5

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bddtests/features/example.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#
```
Comment (rolsonquadras) [inline test/bddtests/features/example.feature]: The file name was changed to e2e in https://github.com/trustbloc/fabric-peer-ext/pull/97#issuecomment-493187027, as this checks for e2e cases.

Comment (fqutishat) [reply test/bddtests/features/example.feature]: done

File: test/bddtests/fixtures/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,301 @@
+#
```
Comment (troyronda) [inline test/bddtests/fixtures/docker-compose.yml]: I think we should keep folders organized. Eg things needs for fabric in its own folder

Comment (fqutishat) [reply test/bddtests/fixtures/docker-compose.yml]: What is the issue of docker-compose.yml

Comment (fqutishat) [reply test/bddtests/fixtures/docker-compose.yml]: done

**Pull request merged**